CHANGELOG.md

```diff
@@ -2,6 +2,72 @@
+- Normalize module path ([a6ce212](https://github.com/serverless/serverless/commit/a6ce212f3d93dbb58236b17dd4052098b57efe3e)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([36da698](https://github.com/serverless/serverless/commit/36da69866ac06608c9274b84e26edb196659fb51)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([01d8960](https://github.com/serverless/serverless/commit/01d89600742b77e35d70f058223e90fe35d942a6)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([c929de0](https://github.com/serverless/serverless/commit/c929de083367feea9b19fe2ac9b38dfce50eca8f)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([657bdea](https://github.com/serverless/serverless/commit/657bdea4bd152dafec8fc27606007a6af437156c)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([a38ed39](https://github.com/serverless/serverless/commit/a38ed390666cb728bacd285517a7e6560a4ad7f7)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([4996e58](https://github.com/serverless/serverless/commit/4996e5869661fae598cefcb64dd008d7948db2ca)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([95a591e](https://github.com/serverless/serverless/commit/95a591e166e2e5d03cb1edb6d0f69f1f7f2b541d)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([38ee5ee](https://github.com/serverless/serverless/commit/38ee5eee50fea4bdc0f1e8baf14bfae8419eae9c)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([b074e35](https://github.com/serverless/serverless/commit/b074e355fca424949484e4811a6be840a4f6c5a2)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([c16dea3](https://github.com/serverless/serverless/commit/c16dea3046459af3af858209fdd185a1016cc338)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([f64b4b2](https://github.com/serverless/serverless/commit/f64b4b25ca799154f69925d35c6d14f3ac2e7e1f)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([6f1c977](https://github.com/serverless/serverless/commit/6f1c97709050e4bbd51a0e14d00990e68eed69e0)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([f81b128](https://github.com/serverless/serverless/commit/f81b12824d4ae19ba9181aa2e611de9756c830cb)) ([Mariusz Nowak](https://github.com/medikoo))
+- Normalize module path ([8d798e3](https://github.com/serverless/serverless/commit/8d798e3427c6c22abdd9ae8614b5cf43990c2b24)) ([Mariusz Nowak](https://github.com/medikoo))
+- Refactor provider-related tests in compile/functions ([#8623](https://github.com/serverless/serverless/issues/8623)) ([07d9602](https://github.com/serverless/serverless/commit/07d9602cd0a2a597a3aa56982cf64d8135b58ee6)) ([Piotr Grzesik](https://github.com/pgrzesik))
```

Comment:
Normally we do not put pure test updates in changelog - they do not affect functionality, and tests are stripped from release.

Still, sometimes I think it's good to recognize community help, for that eventually I would put it into additional "Test improvements" section (your call :)